cms/static/js/certificates/collections/certificates.js

```diff
@@ -66,13 +61,13 @@ function(Backbone, gettext, Certificate) {
 
         parse: function(certificatesJson) {
             // Transforms the provided JSON into a Certificates collection
-            var modelArray = this.certificate_array(certificatesJson);
-
-            for (var i in modelArray) {
-                if (modelArray.hasOwnProperty(i)) {
-                    this.push(modelArray[i]);
+            var modelArray = this.certificateArray(certificatesJson);
+            modelArray.map(function(item) {
```

Comment:
This is actually just `modelArray.filter(item => modelArray.hasOwnProperty(item))`

Reply:
So, why even do something like this in the first place?